Fix: Update to match new Okta UI (#12344)
newrelic/docs-website: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more.mdx

       1. Go to [okta.com/](https://www.okta.com/) and sign in with an account that has administrator permissions.
       2. From the Okta home page, click on **Admin**.
       3. From the Okta admin **Dashboard**, choose the **Applications** page.

>        4. Click **Browse app catalog** and search and select "New Relic by organization".

Reviewer: ```suggestion
       4. Click **Browse app catalog**, then search and select "New Relic by organization".
```

>        4. Click **Browse app catalog** and search for "New Relic by organization" (not "New Relic by account") and choose that from the results.

Reviewer: ```suggestion
       4. Click **Browse app catalog** and search and select "New Relic by organization".
```

       5. From the New Relic by Organization page, click on **Add**.
       6. From the Add New Relic by Organization page, check the two **Application visibility "Do not display..."** checkboxes and click on **Done**. We will make the application visible later after configuration is complete and provisioning has begun.

>        7. Open the newly created application in Okta and go to the **Assignments** tab. This is where you can add users and groups for Authentication.

Reviewer: ```suggestion
       7. Open the newly created application in Okta and go to the **Assignments** tab. This is where you can add users and groups for authentication.
```

>        8. From there, go to the **Sign On** tab. Under **Advanced Sign-on Settings**, you will see **Authentication Domain ID**, you will need to edit this field in step 9 of the general instructions below.  To the right you can click on **View SAML setup instructions**. Under step 7 of these instructions, you can find the URLs required for steps 6 & 7 in the general instructions below.

Reviewer: ```suggestion
       8. From there, go to the **Sign On** tab. Under **Advanced Sign-on Settings**, you'll see **Authentication Domain ID**, which you'll need to edit this field in Step 9 of the general instructions below. To the right, click **View SAML setup instructions**. Under **Step 7** of these instructions, you can find the URLs required for Steps 6 & 7 in the general instructions below.
```
